topo_processor/metadata/loaders/metadata_loader_repo.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+from linz_logger import get_log
+
+from topo_processor.metadata.item import Item
+from topo_processor.util.time import time_in_ms
+
+from .metadata_loader import MetadataLoader
+
+
+class MetadataLoaderRepository:
+    loaders: List[MetadataLoader] = []
+
+    def append_loader(self, loader: MetadataLoader) -> None:
```

Comment:
you generally dont add the name of the thing to the repository, as the repository is already telling you its a  "MetadataLoader"

so you could just do `append`